supplementary_style_guide/style_guidelines/release-notes.adoc

```diff
@@ -63,108 +63,433 @@ Introduce each release note with a heading that summarizes the release note. Thi
 * Do not start the heading with a gerund. Use gerunds only for procedural content.
 
-== Release note categories
+== Release note types and sections
 
-You can write doc texts in more than one way, depending on the issue type:
+Each release note is defined by a specific type based on the information it provides to customers. In Jira tickets, the type is defined in the *Release Note Type* field.
 
-* Enhancement
-* Bug fix
-* Known issue
-* Technology Preview
-* Deprecated functionality
+In a release note document, each release note type is presented in a specific section. Do not use other section names for these release note types.
 
-Use the following general structure for consistency.
+.Release note types and sections
+|===
+|Release note type |Release note section
 
-=== Enhancement
+|Feature, Enhancement, Rebase |New features and enhancements
+|Technology Preview |Technology Preview features
+|Deprecated functionality |Deprecated features
+|Removed functionality |Removed features
+|Known issue |Known issues
+|Bug fix |Fixed issues
+|===
 
-Use present tense in one of the following formats:
-----
-This enhancement <present tense explanation>.
-----
-----
-With this update, <present tense explanation>.
-----
-.Example doc texts
+Every release note type has a template, which is pre-filled in many Jira projects, and that engineers fill in to provide the required information. The writer then rewrites that information into a customer-readable *release note text* (RN text). You can use standard connecting phrases, for example, “As a result,” for results. Sometimes, the information is better presented by changing the order of the pieces of information, for example, a consequence before the cause, or combining them into a single sentence.
+
+=== New features and enhancements
+
+New features are new functions, and enhancements are improvements to existing functions. The release notes for both types are similar, and you can group them together in a single section, or they can be separate.
+
+.New feature and enhancement engineering template
 ----
-This enhancement optimizes migration of an RBD volume from one Cinder back end to another when the volume resides within the same Ceph cluster. If both volumes are in the same Ceph cluster, Ceph performs data migration instead of the cinder-volume process. This reduces migration time.
+Feature, enhancement – describe the feature or enhancement from the user's point of view
+Reason – why has the feature or enhancement been implemented
+Result – what is the current user experience
 ----
+
+.New feature and enhancement release note text
+====
+_<Heading that summarizes the enhancement or feature>_::
+_<Feature, enhancement>_. _<Reason>_. As a result, _<result>_.
++
+For more information, see _<link_to_product_docs>_.
```

Comment:
Might be worth keeping for a later discussion point, but the DITA migration plan mentions stripping out lead-in sentences for links.